lib/screens/study_nurse_screen/configuration_screen/pages/configuration_form.dart

```diff
@@ -154,12 +155,12 @@ class _ConfigurationFormState extends State<ConfigurationForm> {
             onChanged: (String? newValue) {
               widget.callbackForm('entryEmail', newValue!);
             },
-            validator: qValidator(
-              <TextValidationRule>[
-                IsRequired(AppLocalizations.of(context)!.entryEmail),
-                IsEmail(AppLocalizations.of(context)!.entryValidEmail),
-              ],
-            ),
+            validator: (String? value) {
+              if (value == null || value.isEmpty) {
+                return AppLocalizations.of(context)!.entryEmail;
+              }
+              return null;
+            },
```

Comment:
Here, you aren't checking for a valid e-mail anymore. Is that intended?

Reply:
No, it wasn't intended. It has been added now.